spotbugsTest fails on Windows systems

When one executes the `spotbugsTest` task on Windows systems, the following happens:
```
$ gradle spotbugsTest
> Task :spotbugsTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':spotbugsTest'.
> A failure occurred while executing com.github.spotbugs.snom.internal.SpotBugsRunnerForWorker$SpotBugsExecutor
   > 79 SpotBugs violations were found. See the report at: file:///C:/***/rhino/buildGradle/reports/spotbugs/test.xml

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 10s
5 actionable tasks: 1 executed, 4 up-to-date
```
And that's due to spotbugs not matching a couple of source exclusions from the configuration (it uses native paths with the backslash `\` separator to compare source paths).

I'm preparing a PR to work around the issue.

--- a/spotbugs-exclude.xml
+++ b/spotbugs-exclude.xml
@@ -6,7 +6,13 @@
     <Source name="~.*\/testsrc\/.*"/>
   </Match>
   <Match>
+    <Source name="~.*\\testsrc\\.*"/>
+  </Match>
+  <Match>
     <Source name="~.*\/examples\/.*"/>
+  </Match>
+  <Match>
+    <Source name="~.*\\examples\\.*"/>
   </Match>
   <Match>
     <Class name="~org\.mozilla\.javascript\.benchmarks\.jmh_generated.*"/>

